app-text/msort/msort-8.53.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit autotools
+
+DESCRIPTION="A program for sorting files in sophisticated ways"
+HOMEPAGE="https://billposer.org/Software/msort.html"
+SRC_URI="https://billposer.org/Software/Downloads/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="heap +icu +libuninum"
+
+RDEPEND="dev-libs/tre
+	icu? ( dev-libs/icu:= )
+	!icu? ( dev-libs/libutf8proc:= )
+	libuninum? ( dev-libs/libuninum:= )"
+
+DEPEND="${RDEPEND}"
+
+PATCHES=(
```

Comment:
Why so many patches? Is this a dead package?

Reply:
I don't think it's dead, it just older ;-) Latest release is from 2010.
All patches except `msort-8.53-find-utf8proc-headers.patch` and `msort-8.53-respect-users-cflags.patch`, which I wrote by myself, are taken and adaptec from debian.

Reply:
Well, I guess it depends on the sight. From your POV, well.. yes?
But it's working fine, even when it's older. Debian still provides it.